core/state-db/src/lib.rs

```diff
@@ -29,23 +29,17 @@
 //! See `RefWindow` for pruning algorithm details. `StateDb` prunes on each canonicalization until pruning
 //! constraints are satisfied.
 
-#[macro_use] extern crate log;
-#[macro_use] extern crate parity_codec_derive;
-extern crate parking_lot;
-extern crate parity_codec as codec;
```

Comment:
As we can not rename `parity_codec` currently in the `Cargo.toml`, I would like to stick to renaming it here.